openedx/core/lib/api/view_utils.py

```diff
@@ -92,7 +92,10 @@ def _decorator(func_or_class):
         Requires either OAuth2 or Session-based authentication.
         If is_user is True, also requires username in URL matches the request user.
         """
+        from edx_rest_framework_extensions.authentication import JwtAuthentication
```

Comment:
@clintonb  
From Here: https://docs.djangoproject.com/en/1.9/ref/applications/#troubleshooting
`Executing database queries with the ORM at import time in models modules will also trigger this exception. The ORM cannot function properly until all models are available.
Another common culprit is django.contrib.auth.get_user_model(). Use the AUTH_USER_MODEL setting to reference the User model at import time.`

This import is happening during django.setup() when the models have not all been created.
Could you point me to a working example of an app that is importing the authentication module in this way?  
I found two apps that import this module in edx-platform but I put a debugger before the import and it never got hit when I ran paver devstack lms --fast, so I suspect they might not be getting used at this time